lib/fastlane/runner.rb

```diff
@@ -124,7 +124,7 @@ def execute_action(method_sym, class_ref, arguments, custom_dir: '..')
       verify_supported_os(method_sym, class_ref)
 
       begin
-        Dir.chdir(custom_dir) do # go up from the fastlane folder, to the project folder
+        #Dir.chdir(custom_dir) do # go up from the fastlane folder, to the project folder
```

Comment:
Incorrect indentation detected (column 8 instead of 10).<br>Missing space after #.